.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
 show-source = true
-max-line-length = 512
+max-line-length = 160
 select=A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z
-max-complexity = 24
+max-complexity = 12
```

Comment:
sure, but i get an annoying email every time the build fails. please only submit patches for merges which succeed for the love of my inbox ;-)